GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -253,6 +255,19 @@ internal FormBrowse(GitUICommands commands, BrowseArguments args, SettingsSource
             _isFileHistoryMode = args.IsFileHistoryMode;
             InitializeComponent();
 
+            _loadingToolStripMenuItem = new()
+            {
+                Text = _loading.Text,
+                Enabled = false
+            };
+
+            // Prevent user to open menus while plugins still not loaded to work around a bug in winforms
+            pluginsToolStripMenuItem.Enabled = false;
+            toolsToolStripMenuItem.Enabled = false;
+            helpToolStripMenuItem.Enabled = false;
+
+            pluginsToolStripMenuItem.DropDownItems.Insert(0, _loadingToolStripMenuItem);
```

Comment:
I have put back this code that I removed just yesterday as normally it was no more necessary due to the work around for a winforms menu bug I was having every time. For a reason I don't explain, I don't have the bug anymore and so I have removed the work around code and so reinstated this code. Sorry for the back and forth.

So now, it makes sense to answer this comment 😅: 
I prefer to keep it like that so that I can reuse a translated string and so don't have to manage one more and also to stay consistent with all the dynamic ToolStripItems of the plugins menus that are all handled in this file.

Reply:
It should be possible to set text to "" or skip text still.
A nit for me too.

sidenote: We should change the translator app to ignore texts starting with `NO_TRANSLATE` or so to avoid prefixing with _NO_TRANSLATE and to have these discussions.

sidenote2: Similar code in FileStatusList. However, there it affects the order of items, There are default items, the "form specific" are handled, then the FileStatusList conditional, so it would affect the order of items (without more changes).

Reply:
Having it defined in the designer and trying to avoid having it translated and replacing the translation feel hacky for me.

So either, I put it in the designer and let it be translated by normal process or I keep it like that.
I will put it in the designer as you seems to both prefer...